test/common/cdp.py

```diff
@@ -170,7 +170,7 @@ def start(self):
 
             # sandboxing does not work in Docker container
             self._browser = subprocess.Popen(
-                argv + ["--disable-gpu", "--no-sandbox", "--remote-debugging-port=%i" % cdp_port, "about:blank"],
+                argv + ["--disable-gpu", "--no-sandbox", "--disable-web-security", "--remote-debugging-port=%i" % cdp_port, "about:blank"],
```

Comment:
This disables Same-Origin Policy, doesn't it? This will hide actual bugs, so if the unit tests need this for some reason, it should be a (non-default) option of the `CDP` class.